apps/moodlenet/templates/services/postgresql/svc.yml.j2

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
should only be created for trashable envs?

Reply:
Nope, it is used even in production.
The endpoint declared in `ep.yml.j2` is related to this service.